cmd/ocm-backplane/root.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/managedJob"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/script"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/status"
+	"github.com/openshift/backplane-cli/cmd/ocm-backplane/console"
```

Comment:
Suggest moving this upper after "github.com/openshift/backplane-cli/cmd/ocm-backplane/cloud" to follow the alphabet order.